birdhouse/default.env

```diff
@@ -44,17 +44,11 @@ or a custom web interface.
 "
 
 export DEFAULT_CONF_DIRS='
-  ./config/proxy
-  ./config/canarie-api
-  ./config/geoserver
-  ./config/finch
-  ./config/raven
-  ./config/hummingbird
-  ./config/thredds
-  ./config/portainer
-  ./config/magpie
-  ./config/twitcher
-  ./config/jupyterhub
+  ./components/proxy
+  ./components/magpie
+  ./components/twitcher
+  ./components/stac
+  ./components/cowbird
```

Comment:
No difference technically, but can you put cowbird directly after magpie/twitcher given how closely related they are feature wise?